actors/evm/src/lib.rs

```diff
@@ -212,16 +227,28 @@ impl EvmContractActor {
         rt.validate_immediate_caller_accept_any()?;
 
         let state: State = rt.state()?;
-        Ok(state.bytecode)
+        if is_dead(rt, &state) {
+            // TODO: to return the "empty bytecode" cid, we'd need to actually write the empty
+            // bytecode. Otherwise, it's not reachable.
+            // Or we could implement https://github.com/filecoin-project/ref-fvm/issues/1358.
+            // Finally, we could just return an error and let the caller deal with it?
```

Comment:
Could we return an Option from here? Both EXTCODE* and JSON-RPC handling would be simple.

Reply:
You know... that's actually nice. It makes it easy to tell if a contract has been selfdestructed.